topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.md

```diff
@@ -0,0 +1,421 @@
+
+> ### {% icon hands_on %} Hands-on: Rename channels
+>
+> 1. {% tool [Rename OME-TIFF Channels](toolshed.g2.bx.psu.edu/repos/watsocam/rename_tiff_channels/rename_tiff_channels/0.0.1.2) %} with the following parameters:
+>
+>    - {% icon param-file %} *"Input image in either tiff or OME-tiff format"*: File to be converted (or collection of files to run in batch)
+>    - *"Format of input image"*: `ome.tiff`
+>    - {% icon param-file %} *"Channel metadata CSV"*: Comma-separated markers file with marker_names in third column
+>
+{: .hands_on}
+
+
+## Initial visualization with **Avivator**
+
+For any `OME-TIFF` image in a Galaxy-MTI history, there will be an option to view the image using **Avivator**. This is a great way to perform an initial inspection of an image for QC purposes before continuing with downstream steps. The **Avivator** window can be launched by expanding the dataset information in the history panel and clicking the link (Figure 7.).
+
+![vivhistory](../../images/multiplex-tissue-imaging-TMA/ex2_avivatorHistory.png "The highlighted link automatically appears for any OME-TIFF image (left) and, when clicked, launches an Avivator window to explore the image (right).")
+
+
+## Generating an interactive visualization dashboard with **Vitessce**
+
+**Vitessce** is a powerful visualization tool that creates interactive dashboards (Figure 8.) to look at a multiplex `OME-TIFF` images in conjunction with data generated during analysis and stored in an anndata file. The segmentation mask can be overlaid onto the image to qualitatively assess the segmentation performance. The mask can then be colored with associated observational data (Figure 9A.), such as `phenotype`, with the same colors appearing in barplots (Figure 9B.), UMAP representations, heatmaps, and marker intensity violin plots for comrehensive data exploration. 
+
+![vitessce](../../images/multiplex-tissue-imaging-TMA/ex2_fullVitessce.png "A Full view of a vitesse dashboard for one core from Exemplar-002.")
+
+![vitessce](../../images/multiplex-tissue-imaging-TMA/ex2_vitessce_zoomed.png "Each window in the dashboard can be resized to view the components in more detail. A closer look at the phenotype-labeled mask overlaid on the actual image (A), and the phenotype barplot (B).")
+
+> ### {% icon hands_on %} Hands-on: Vitessce visualization
+>
+> 1. {% tool [Vitessce Visualization](vitessce_spatial) %} with the following parameters:
+>
+>    - {% icon param-file %} *"Select the OME Tiff image"*: OME-TIFF image to be viewed (or collection of files to run in batch)
+>    - {% icon param-file %} *"Select masks for the OME Tiff image (Optional)"*: Output of Mesmer (or other segmentation tool)
+>    - *"Whether to do phenotyping"*: `Yes`
```

Comment:
Do I need to select an `AnnData` file here? If so it seems two flies are compatible (I'm guessing "convert to anndata" though)